pkg/actuators/machine/utils.go

```diff
@@ -398,3 +403,97 @@ func GetMibValueOfQuantity(quantity resource.Quantity) int64 {
 func Ptr[T any](v T) *T {
 	return &v
 }
+
+// GetGPU returns the VMGpu for the given Prism Element (uuid) and GPU identifier input
+func GetGPU(ctx context.Context, client *nutanixClientV3.Client, peUUID string, gpu machinev1.NutanixGPU) (*nutanixClientV3.VMGpu, error) {
```

Comment:
Does this need to be a public method?

Reply:
Changed the method name to start with lower case.